static/usage/datetime/theming/index.md

```diff
@@ -7,23 +7,24 @@ import vue from './vue.md';
 import angularHTML from './angular-html.md';
 import angularCSS from './angular-css.md';
 
-<Playground 
-  size="450px" 
-  code={{ 
-    javascript, 
+<Playground
+  size="450px"
+  code={{
+    javascript,
     react: {
       files: {
         'src/main.tsx': reactHTML,
-        'src/main.css': reactCSS
-      }
-    }, 
-    vue, 
+        'src/main.css': reactCSS,
+      },
+    },
+    vue,
     angular: {
       files: {
-        'src/app/app.component.html': angularHTML,
-        'src/styles.css': angularCSS
-      }
-    }
-  }} 
+        'src/app/example.component.html': angularHTML,
+        'src/styles.css': angularCSS,
+      },
+    },
+  }}
   src="usage/datetime/theming/demo.html"
+  includeIonContent={false}
```

Comment:
Why do we not include ion-content here? Also I noticed that ion-app does not get injected in either Angular or JS examples.